app/assets/stylesheets/views/_service-manual-guide.scss

```diff
@@ -0,0 +1,145 @@
+// Proposed component:'govuk_component/breadcrumb'
+
+.govuk-breadcrumb {
+  ol {
+    @extend %contain-floats;
+    padding-top: 0.75em;
+    padding-bottom: 0.75em;
+    li {
+      @include core-16;
+      float: left;
+      background-image: file-url("separator.png");
+      @include device-pixel-ratio() {
+        background-image: file-url("separator-2x.png");
+        background-size: 6px 11px;
+      }
+      background-position: 100% 50%;
+      background-repeat: no-repeat;
+      list-style: none;
+      margin-right: 0.5em;
+      margin-bottom: 0.4em;
+      margin-left: 0;
+      padding-right: 1em;
+      a {
+        color: $text-colour;
+      }
+      strong {
+        font-weight: normal;
+      }
+      &:last-child {
+        background-image: none;
+        margin-right: 0
+      }
+    }
+  }
+}
+
+
+// Modify existing component: 'govuk_component/title'
+
+.govuk-title {
+  margin-top: 1.875em; // 30/16
```

Comment:
Absolutely, I was just trying to make it clear where the em values had come from.

Ideally, I'd like to use a [px-to-em function, like the one in elements](https://github.com/alphagov/govuk_elements/blob/master/public/sass/elements/_helpers.scss#L16), I'll create a PR to add this to the front end toolkit.

// Return ems from a pixel value
@function em($px, $base: 19) {
  @return ($px / $base) + em;
}

So instead it would be `margin-top: em(30,16);`